package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@morpho-org/morpho-blue",
   "description": "Morpho Blue Protocol",
   "license": "BUSL-1.1",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
It's the tag we used for the release right ? So should we update the release tag as well ?

Reply:
I'm not against @MerlinEgalite ?

Reply:
I'd prefer to have released correlated to contract changes but since morpho blue won't change maybe it's not that relevant. I don't mind about both options tbh

Reply:
the tag is `v1.0.0` what's the problem?

Reply:
> the tag is v1.0.0 what's the problem?

When this PR will be merged, the tag 1.0.0 will still have `0.1.0` in the `version` field of `package.json`, which I find confusing

Reply:
got it
yes let's update